msteams-platform/concepts/deploy-and-publish/appsource/prepare/frequently-failed-cases.md

```diff
@@ -1,246 +1,348 @@
+* Microsoft Windows 7 and later
+* macOS 10.10 and later
+* Microsoft Edge 12 and later
+* Mozilla Firefox 47.0 and later
+* Google Chrome 51.0 and later
+* iOS 9.0 and later
+* Android 4.4 and later
 
-> [!div class="checklist"]
->
-> * within your app's description section.
-> * in your app's welcome message.
-> * in your app's help message.
-> * in the window where you ask a user to sign-in to your services.
+### 3.3 Response time
 
-* Apps without an easy sign-up flow must also include a help tab or link to a web page, where a new user can see detailed guidance on configuring your Teams app. Provide detailed information to ensure a new user is not blocked when trying your app for the first time.
-* Sign-in and sign-out functionality must work on mobile clients. Ensure to use the [Microsoft Teams SDK](https://www.npmjs.com/package/@microsoft/teams-js) version 1.4.1 or later.
+Teams apps must respond within a reasonable timeframe, which varies depending on the capability.
 
-For additional information on authentication see:
+* Tabs must respond within three seconds or display a loading message or warning.
+* Bots must respond to user commands within two seconds or display a typing indicator.
+* Messaging extensions must respond to user commands within five seconds.
+* Notification must load within five seconds of the user action.
 
-* [Authentication documentation](../../../authentication/authentication.md)
-* [Bot authentication sample in Node](https://github.com/OfficeDev/microsoft-teams-sample-auth-node)
-* [Tab authentication sample in Node](https://github.com/OfficeDev/microsoft-teams-sample-complete-node)
-* [Tab/bot authentication in C#/.NET](https://github.com/OfficeDev/microsoft-teams-sample-complete-csharp)
+## 4.0 App package and store listing
 
-### &#9989; Response times must be reasonable
+App packages must be correctly formatted and include all required information and components.
 
-* **Tabs**. If a response to an action takes more than three seconds, you must provide a loading message or warning.
-* **Bots**. A response to a user command must occur within two seconds. If longer processing is required, your app must display a typing indicator.
-* **Compose extensions**. A response to a user command must occur within five seconds.
+### 4.1 App manifest
+
+* The app manifest must conform to the [latest manifest schema](~/resources/schema/manifest-schema.md).
+* If your app includes a bot or messaging extension, the manifest must be consistent with Bot Framework metadata, including bot name, logo, privacy policy link, and terms of service link.
+* If your app uses Azure Active Directory (Azure AD) for authentication, include the Azure AD ID in the manifest. For more information, see [manifest schema](~/resources/schema/manifest-schema.md#webapplicationinfo).
+
+### 4.2 App icons
```

Comment:
@ramendhe I wonder if app icons, app descriptions, and screenshots sections belong under 1.0 Value proposition along with the app name section?

Reply:
@heath-hamilton, I would prefer to keep it under store listing as it would be too much details to put under value proposition